contracts/utils/SafeOwnable.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.5.17;
+
+/**
+ * Based on OpenZeppelin's Ownable contract:
+ * https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol
+ *
+ * @dev Contract module which provides a basic access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions.
+ *
+ * This module is used through inheritance. It will make available the modifier
+ * `onlyOwner`, which can be applied to your functions to restrict their use to
+ * the owner.
+ */
+contract SafeOwnable {
```

Comment:
why is this contract added? i do not see it in use anywhere.
also, i'm a bit concerned about adding an ownable contract with a different interface than the once we're using elsewhere. makes things unnecessary complicated.

Reply:
@tjcloa

Reply:
i think adjusting the contract to be compatible with our existing interfaces is nothing we need to do now without a use-case for it. I would remove the contract for now to not create confusion and when we add a smart contract from scratch the next time, we can start using it.

Reply:
removing it